packages/grid/test/column-reordering.test.js

```diff
@@ -59,7 +59,7 @@ describe('reordering simple grid', () => {
 
   beforeEach(async () => {
     grid = fixtureSync(`
-      <vaadin-grid style="width: 400px; height: 200px;" size="1" column-reordering-allowed>
+      <vaadin-grid style="width: 450px; height: 200px;" size="1" column-reordering-allowed>
```

Comment:
This test failed in WebKit on Linux because for footer cells, `grid._cellFromPoint(x, y)` returned `<table>` element instead of the actual cell. It was most likely caused by presence of horizontal scroll bar:

![Screenshot 2022-08-10 at 16 58 19](https://user-images.githubusercontent.com/10589913/183921209-d57161f0-504c-455f-bec4-c0c4e38b6f8e.png)

Increased the width to get rid of the scrollbar in this test suite.